vizro-core/docs/pages/user-guides/actions.md

```diff
@@ -104,6 +100,16 @@ Therefore, the exported data will reflect any native filters and parameters, but
 
     [Graph]: ../../assets/user_guides/actions/actions_export.png
 
+
+!!! note
+
+    Note that exported data will not include any chart modifications, such as filtering, sorting, or disabling chart traces from the legend.
+    Only the original data_frame input, with any native filters and parameters applied, is exported.
+
+    When exporting data from a [custom chart](custom-charts.md), it is the contents of the data_frame input argument that is exported.
+    As a result, the exported data will include any native filters and parameters, but will not reflect any transformations applied to the data_frame within the chart function.
```

Comment:
I find this a bit confusing and would simplify it. Please confirm if the below is correct (as this is my current understanding). I don't think parameters have any effect on the original data source? They change a dimension in the chart but they don't change the data source, so I didn't mention them below. But let me know if you have a different understanding 👍 

```suggestion
    Note that exported data will only reflect the original data set with all filters defined with `vm.Filter` applied. Filters from the chart itself (e.g., ag-grid filters), other chart modifications, and any data transformations in custom charts are not included.
```

Reply:
```suggestion
    Note that exported data only reflects the original dataset and any native filters defined with `vm.Filter`. Filters from the chart itself, such as ag-grid filters, are not included, and neither are other chart modifications, nor any data transformations in custom charts.
```

Reply:
I've made just a few minor suggestions onto yours @huong-li-nguyen.

Reply:
Thanks for the review 🙌 

I understand that these phrases need to be simplified, but it seems we've oversimplified them 😄. You have perfectly described which modifications and transformations do not affect the export data. However, there is a lack of info about what affects exported data, as exported data could be affected by:
 - vm.Filter (filter action)
 - filter_interaction action
 - vm.Parameter (if it parametrises the dynamic data function argument)

So what do you think about this suggestion:
```suggestion
    Note that exported data only reflects the original dataset and any native data modifications defined with [`vm.Filter`](filters.md), [`vm.Parameter`](data.md/#parametrize-data-loading) or [`filter_interaction`](actions.md/#filter-data-by-clicking-on-chart) action. 
    Filters from the chart itself, such as ag-grid filters, are not included, and neither are other chart modifications, nor any data transformations in custom charts.
```

Reply:
This looks good to me!